src/main/java/gyro/aws/dynamodb/DynamoDbAttributeDefinition.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.dynamodb;
+
+import gyro.aws.Copyable;
+import gyro.core.resource.Diffable;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidStrings;
+import software.amazon.awssdk.services.dynamodb.model.AttributeDefinition;
+
+public class DynamoDbAttributeDefinition extends Diffable implements Copyable<AttributeDefinition> {
+
+    private String name;
+    private String type;
+
+    /**
+     * The name for the attribute. (Required)
+     */
+    @Required
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * The data type for the attribute. Valid values are ``S`` (string), ``N`` (number), or ``B`` (binary). (Required)
+     */
+    @Required
+    @ValidStrings({ "S", "N", "B" })
```

Comment:
`attribute` field of the `dynamodb-table` resource is updatable. Now any change to the `name` field of the attribute definition would trigger a delete and create as it is the primary key, but a change of type results in replace workflow

```
Looking for changes...

⟳ Update aws::dynamodb-table dynamodb-table-example (dynamodb-table-example)
· attribute:  ⟳ attribute RangeKeyName → attribute HashKeyName, ⟳ attribute GlobalRangeKeyName → attribute LocalRangeKeyName, ⟳ attribute HashKeyName → attribute GlobalRangeKeyName, ⟳ attribute LocalRangeKeyName → attribute RangeKeyName

    ⇅ Replace attribute HashKeyName (because of type, skipping without a workflow)
    · type:  'S' → 'N'


Are you sure you want to change resources? (y/N)
```
```suggestion
    @Updatable
    @ValidStrings({ "S", "N", "B" })
```

Reply:
Attributes cannot be updated once they have been created. The only scenario where attributes can be updated is when creating a new LSI/GSI, new attributes can be defined.